src/libasr/codegen/asr_to_c.cpp

```diff
@@ -415,6 +415,11 @@ class ASRToCVisitor : public BaseCCPPVisitor<ASRToCVisitor>
                 headers.insert("complex.h");
                 convert_variable_decl_util(v, is_array, declare_as_constant, use_ref, dummy,
                     force_declare, force_declare_name, n_dims, m_dims, v_m_type, dims, sub);
+            } else if (ASR::is_a<ASR::SymbolicExpression_t>(*v_m_type)) {
+                headers.insert("symengine/cwrapper.h");
+                std::string type_name = "basic";
+                std::string v_m_name = v.m_name;
+                sub = format_type_c("", type_name, v_m_name, use_ref, dummy);
```

Comment:
These 4 lines can be reverted. They were introduced in this PR (https://github.com/lcompilers/lpython/pull/1846) and hence might have been introduced in LFortran through the previous sync but these changes were further reverted here https://github.com/lcompilers/lpython/pull/2260